### PR TITLE
[eclipse/xtext-core#69] Added test language and test case.

### PR DESCRIPTION
**Pull request by ArneDeutsch**
Primary artifacts (everything else is generated):
- ContentAssistNoTerminalExtensionTestLanguage.xtext
- ContentAssistNoTerminalExtensionContentAssistTest.java

Signed-off-by: Arne Deutsch <Arne.Deutsch@itemis.de>

### CODE REVIEW

**Comment by ArneDeutsch** [comment]
The tests will siucceed of course only if https://github.com/eclipse/xtext-core/pull/776 is merged before.

File: org.eclipse.xtext.ui.tests/tests/org/eclipse/xtext/ui/tests/editor/contentassist/ContentAssistNoTerminalExtensionContentAssistTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ * Tests introduced for https://github.com/eclipse/xtext-core/issues/69.
+ * 
+ * @author Arne Deutsch - Initial contribution and API
+ */
+@InjectWith(ContentAssistNoTerminalExtensionTestLanguageUiInjectorProvider.class)
+@RunWith(XtextRunner.class)
+public class ContentAssistNoTerminalExtensionContentAssistTest extends AbstractContentAssistTest {
+
+	@Test
+	public void expectedAutocomplationAfterTwoCharacters() throws Exception {
```
Comment (szarnekow) [inline org.eclipse.xtext.ui.tests/tests/org/eclipse/xtext/ui/tests/editor/contentassist/ContentAssistNoTerminalExtensionContentAssistTest.java]: Typo in method name

**Comment by ArneDeutsch** [comment]
Fixed typo.

**Comment by cdietrich** [comment]
@ArneDeutsch @szarnekow what is left for this pr?

**Comment by ArneDeutsch** [comment]
If @szarnekow agrees I would merge this PR. Nothing left.

**Pull request merged**